### DISCUSSION TITLE
Export & Import Library

### DISCUSSION THREAD

**Discussion by ghost** [ciderapp/Cider#1027] [Feature Request]
Currently on iTunes, you can export & import your entire library with the click of a button into a single file. This file could then be used to, theoretically, transfer all of your music between accounts or be used to simply just back your library up. 
It would be amazing if this feature was imported to Cider, it'd mean that you could keep a safe copy of your Apple Music library, and if anything were to go wrong, i.e. your library were to get corrupted, you could simply restore your library to one of these backups.

**Comment by mrPsycho** [comment]
agree, it also may help with changing region.
